scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -53,8 +53,8 @@
             -ReleaseLabel "Release-ARM-1"
             -ArtifactImageBuildNumber "20190101.1"
             -TestName "TempSensor"
-            -ContainerRegistry "edgebuilds.azurecr.io"
-            -ContainerRegistryUsername "EdgeBuilds"
+            -ContainerRegistry "yourpipeline.azurecr.io"
```

Comment:
Nit: The container registry doesn't have to be hosted on azurecr.io. It could be dockerhub, or your own hosted server, or anything else.

Reply:
Updated the parameter comment. please re-approve.